modules/apps/document-library/document-library-video/src/main/java/com/liferay/document/library/video/internal/helper/DLVideoExternalShortcutDLFileEntryTypeHelper.java

```diff
@@ -73,6 +72,8 @@ public void addDLVideoExternalShortcutDLFileEntryType() throws Exception {
 			_addDLVideoExternalShortcutDLFileEntryType(
 				ddmStructure.getStructureId());
 		}
+
+		_updateDLFileEntryTypeNameMap(dlFileEntryType);
```

Comment:
Same things I commented for the Google shortcut apply here: method rename and else clause.